src/internal/log/loggers.go

```diff
@@ -214,12 +214,12 @@ func InitBatchLogger(logFile string) func(err error) {
 			if logFile == "" {
 				var err error
 				// On error, out is set to "" again.
-				logFile, err = xdg.CacheFile(fmt.Sprintf("pachyderm/log/%s.%s.log", name, time.Now().In(time.UTC).Format("20060102T150405Z")))
+				name := fmt.Sprintf("%s.%s.log", name, time.Now().In(time.UTC).Format("20060102T150405Z"))
```

Comment:
Not allowed to write to ~/.cache in a genrule.

Reply:
Could you set `XDG_CACHE_HOME` when this is called from a genrule?

Reply:
It seems like we can't.  Overall, I like having the log when "all else fails" even if you have to delete spurious files ("git clean" saves you).